src/Graphics.cpp

```diff
@@ -288,13 +289,21 @@ void Gosu::Graphics::transform(const Gosu::Transform& transform, const function<
 }
 
 void Gosu::Graphics::draw_line(double x1, double y1, Color c1,
-    double x2, double y2, Color c2, ZPos z, AlphaMode mode)
+    double x2, double y2, Color c2, ZPos z, AlphaMode mode, double thickness)
 {
     DrawOp op;
     op.render_state.mode = mode;
-    op.vertices_or_block_index = 2;
-    op.vertices[0] = DrawOp::Vertex(x1, y1, c1);
-    op.vertices[1] = DrawOp::Vertex(x2, y2, c2);
+    op.vertices_or_block_index = 4;
+
+    double angle = Gosu::angle(x1, y1, x2, y2)-90.0;
```

Comment:
+space around `-`

Reply:
Also, you don't need to use `Gosu::` inside a Gosu method (I've started to clean this up recently, I used to be undecided about this)

Reply:
And you can ignore the call if `x1 == x2 && y1 == y2`, in which case `Gosu::angle` cannot return anything useful.